doc/whats_new/v1.0.rst

```diff
@@ -512,6 +512,14 @@ Changelog
   precision of the computed variance was very poor when the real variance is
   exactly zero. :pr:`19766` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+:mod:`sklearn.discriminant_analysis`
+....................................
+
+- |Enhancement| :class:`discriminant_analysis.QuadraticDiscriminantAnalysis` 
+  will now cause 'RuntimeWarning' in case of collinear variables. These errors
+  can be silenced by the 'reg_param' attribute.:pr:`19731` by 
+  :user:`Alihan Zihna <azihna>`
```

Comment:
May we move the `discriminant_analysis` section higher in this list? (The modules are in alphabetical order)

```suggestion
- |Enhancement| :class:`discriminant_analysis.QuadraticDiscriminantAnalysis` 
  will now cause 'LinAlgWarning' in case of collinear variables. These errors
  can be silenced by the 'reg_param' attribute. :pr:`19731` by 
  :user:`Alihan Zihna <azihna>`
```